apps/condo/domains/ticket/schema/TicketAnalyticsReportService.js

```diff
@@ -21,55 +47,60 @@ const TicketAnalyticsReportService = new GQLCustomSchema('TicketAnalyticsReportS
         },
         {
             access: true,
-            type: 'type TicketAnalyticsReportOutput { result: JSON! }',
+            type: 'type TicketAnalyticsReportOutput { result: [TicketGroupedCounter!] }',
```

Comment:
You will have `data.result.result` in output, let it be `data.result.groups`, for example.

```suggestion
            type: 'type TicketAnalyticsReportOutput { groups: [TicketGroupedCounter!] }',
```

Reply:
Done